scripts/build-abc.sh

```diff
@@ -20,6 +20,7 @@ case "$OS" in
     case "$ARCH" in
       "I386") A="-m32 -DLIN -DSIZEOF_VOID_P=4 -DSIZEOF_LONG=4 -DSIZEOF_INT=4" ;;
       "X86_64") A="-m64 -fPIC -DLIN64 -DSIZEOF_VOID_P=8 -DSIZEOF_LONG=8 -DSIZEOF_INT=4" ;;
+      "AArch64") A="-m64 -fPIC -DLIN64 -DSIZEOF_VOID_P=8 -DSIZEOF_LONG=8 -DSIZEOF_INT=4" ;;
```

Comment:
`-DLIN64` looks suspicious if you are targeting MacOS, but I'm good with this if you are.

Reply:
I thought so too! But interestingly, that is what `abc` itself reports. Specifically, if you compile `abc`'s [`arch_flags`](https://github.com/berkeley-abc/abc/blob/4656ae10e08a086d111e6982f057d18babad1150/arch_flags.c) script and run it on Apple silicon, then it reports the following:

```
$ ./arch_flags
-DLIN64 -DSIZEOF_VOID_P=8 -DSIZEOF_LONG=8 -DSIZEOF_INT=4
```

If `abc` itself recommends `-DLIN64`, then I'm happy to oblige :)